### ISSUE TITLE
Collections without Concepts are not exported

### ISSUE THREAD

**Issue by KlausIllmayer** [acdh-oeaw/vocabseditor#46]
Currently if I export a ConceptScheme that contains Collections without Concepts, such Collections will not show up in the dump (one example is our [VICAV](https://vocabseditor.acdh.oeaw.ac.at/vocabs/scheme/368). It could be that skosify kicks them out. Just to be sure, is this a behavior built into vocabseditor or is it indeed skosify that does this clean-up?

tbd: If vocabseditor is ignoring such Collections, then I tend to put them into the export. If skosify is the reason, then we accept it.

**Comment by csae8092** [comment]
The main entity for the export is the SkocConcept (process all concepts related to the given schema); thus collections without any related concepts are ignored (see e.g. https://github.com/acdh-oeaw/vocabseditor/blob/master/vocabs/rdf_utils.py#L98)
we can of course change this but whats the use of collection with no concepts?
@KlausIllmayer please decide if I should include those collections

**Comment by KlausIllmayer** [comment]
Indeed, vocabularies in production should not have Collections without Concepts. But in case of vocabularies in development it may make sense. I can imagine that you point curators/reviewers to a bundle of Collections for comments and to add new Concepts based on an initial set of Collections. Usually this should happen in the vocabseditor but it could be, that external users don't use vocabseditor and instead only have access to the Skosmos view (where I guess, that such Collections show up). I don't think that it is a frequent workflow, but who knows. In general, I would pass over such a decision to Skosify. So if it is only a minor change on your side, please include those collections -(otherwise lets leave it as it is now and have it documented).
